src/service_main.rs

```diff
@@ -25,19 +25,33 @@ extern crate serde;
 #[macro_use]
 extern crate serde_derive;
 extern crate serde_json;
+extern crate failure;
+extern crate mozsvc_common;
+extern crate slog_async;
+extern crate slog_mozlog_json;
+extern crate slog_term;
+#[macro_use(
+   slog_o, slog_info, slog_kv, slog_log, slog_record, slog_b, slog_record_static, slog_error
+)]
+extern crate slog;
```

Comment:
strange sort order here, the main slog got put all the way to the bottom. *Shrugs*